pkg/cmd/operate.go

```diff
@@ -43,11 +43,28 @@ func operate(provider, arguments string) {
 
 	gardenClientset, err := target.GardenerClient()
 	checkError(err)
-
-	project, err := gardenClientset.CoreV1beta1().Projects().Get(target.Stack()[1].Name, metav1.GetOptions{})
-	checkError(err)
-	shoot, err := gardenClientset.CoreV1beta1().Shoots(*project.Spec.Namespace).Get(target.Stack()[2].Name, metav1.GetOptions{})
-	checkError(err)
+	var shoot *gardencorev1beta1.Shoot
+	if target.Stack()[1].Kind == "project" {
+		project, err := gardenClientset.CoreV1beta1().Projects().Get(target.Stack()[1].Name, metav1.GetOptions{})
+		checkError(err)
+		shoot, err = gardenClientset.CoreV1beta1().Shoots(*project.Spec.Namespace).Get(target.Stack()[2].Name, metav1.GetOptions{})
+		checkError(err)
+	} else if target.Stack()[1].Kind == "seed" {
+		shootList, err := gardenClientset.CoreV1beta1().Shoots("").List(metav1.ListOptions{})
+		checkError(err)
+		var filteredShoots []gardencorev1beta1.Shoot
+		for _, s := range shootList.Items {
+			if s.Name == target.Stack()[2].Name {
```

Comment:
@neo-liang-sap  sorry for beeing late for the review, but here you are missing the check for the seed. Do not add to `filteredShoots` if the seed does not match the one on the target stack

Reply:
thanks! i added a new PR for this [here](https://github.com/gardener/gardenctl/pull/298)